content/de/docs/concepts/overview/what-is-kubernetes.md

```diff
@@ -0,0 +1,176 @@
+konzipiert, um die Bereitstellung, Skalierung und Verwaltung von Anwendungen zu erleichtern.
+
+[Labels](/docs/concepts/overview/working-with-objects/labels/) ermöglichen es den Benutzern, ihre Ressourcen 
+nach Belieben zu organisieren. [Anmerkungen](/docs/concepts/overview/working-with-objects/annotations/) ermöglichen es Benutzern,
+Ressourcen mit benutzerdefinierten Informationen zu versehen, um ihre Arbeitsabläufe zu vereinfachen und eine einfache Möglichkeit
+für Managementtools zu bieten, den Status von Kontrollpunkten zu ermitteln.
+
+
+Darüber hinaus basiert die [Kubernetes-Steuerungsebene](/docs/concepts/overview/components/) auf den gleichen APIs,
+die Entwicklern und Anwendern zur Verfügung stehen. Benutzer können ihre eigenen Controller, wie z.B. 
+[Scheduler](https://github.com/kubernetes/community/blob/{{< param "githubbranch" >}}/contributors/devel/scheduler.md), mit 
+ihren [eigenen APIs](](/docs/concepts/api-extension/custom-resources/) schreiben, die von einem 
+universellen [Befehlszeilenwerkzeug](/docs/user-guide/kubectl-overview/) angesprochen werden können.
+
+Dieses [Design](https://git.k8s.io/community/contributors/design-proposals/architecture/architecture.md) hat es einer Reihe anderer Systeme ermöglicht, auf Kubernetes aufzubauen.
+
+## Was Kubernetes nicht ist
+
+Kubernetes ist kein traditionelles, allumfassendes PaaS (Plattform als ein Service) System. Da Kubernetes nicht auf Hardware-, 
+sondern auf Containerebene arbeitet, bietet es einige allgemein anwendbare Funktionen, die PaaS-Angeboten gemeinsam sind, 
+wie Bereitstellung, Skalierung, Lastausgleich, Protokollierung und Überwachung. 
+Kubernetes ist jedoch nicht monolithisch, und diese Standardlösungen sind optional und modular etweiterbar. 
+Kubernetes liefert die Bausteine für den Aufbau von Entwicklerplattformen, bewahrt aber die 
+Wahlmöglichkeiten und Flexibilität der Benutzer, wo es wichtig ist.
+
+Kubernetes:
+
+* Schränkt nicht die Art der unterstützten Anwendungen ein. Kubernetes zielt darauf ab, 
+  eine extrem große Vielfalt von Workloads zu unterstützen, einschließlich stateless, 
+  stateful und datenverarbeitender Workloads. Wenn eine Anwendung in einem Container ausgeführt 
+  werden kann, sollte sie auf Kubernetes hervorragend laufen.
+* Verteilt keinen Quellcode und entwickelt Ihre Anwendung nicht. 
+  Kontinuierliche Integrations-, Liefer- und Bereitstellungs-Workflows (CI/CD) werden durch 
+  Unternehmenskulturen und -präferenzen sowie technische Anforderungen bestimmt.
+* Does not provide application-level services, such as middleware
+  (e.g., message buses), data-processing frameworks (for example,
+  Spark), databases (e.g., mysql), caches, nor cluster storage systems (e.g.,
+  Ceph) as built-in services. Such components can run on Kubernetes, and/or
+  can be accessed by applications running on Kubernetes through portable
+  mechanisms, such as the Open Service Broker.
```

Comment:
I think this point can be removed as the translation is below

Reply:
Indeed, I forgot to delete the English part